index.d.ts

```diff
@@ -0,0 +1,112 @@
+/** Declaration file generated by dts-gen */
+
+export = dog_statsy;
+
+declare class dog_statsy {
+    constructor(...args: any[]);
+
+    close(...args: any[]): void;
+
+    count(...args: any[]): void;
```

Comment:
I think most (if not all) of these methods shouldn't be using `any` because they have pretty well defined types and don't accept an infinite number of parameters. For example the type for the `count()` method should be:
```ts
count(name: string, val: number, sample: number | null | undefined, tags: string[]): void
```